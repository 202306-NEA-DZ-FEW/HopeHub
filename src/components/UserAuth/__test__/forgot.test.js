import renderer from "react-test-renderer";

import Forgot from "../Forgot";

it("renders correctly", () => {
    const tree = renderer.create(<Forgot />).toJSON();
    expect(tree).toMatchSnapshot();
});
