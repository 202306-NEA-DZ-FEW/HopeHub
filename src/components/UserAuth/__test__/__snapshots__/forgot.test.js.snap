// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="font-poppins flex flex-col items-start justify-center w-full "
>
  <h2
    className=" font-extrabold text-4xl text-NeutralBlack  my-5"
  >
     
    forgot
     
  </h2>
  <form
    className="text-4xl p-6 w-full bg-NeutralWhite dark:bg-Dark_Primary rounded-md shadow-md flex flex-col gap-4"
    onSubmit={[Function]}
  >
    <button
      className="flex flex-row justify-start items-center text-lg gap-2 font-bold  text-Accent "
      onClick={[Function]}
      type=""
    >
       
      <svg
        fill="none"
        height="1em"
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        style={
          Object {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <line
          x1="19"
          x2="5"
          y1="12"
          y2="12"
        />
        <polyline
          points="12 19 5 12 12 5"
        />
      </svg>
       
      Back
       
    </button>
    <input
      className="input input-bordered w-full bg-NeutralWhite text-NeutralBlack h-16"
      onChange={[Function]}
      placeholder="footer_msg_3"
      type="email"
      value=""
    />
    <button
      className="btn bg-Accent text-NeutralBlack dark:text-NeutralWhite  dark:bg-Dark_Accent dark:hover:bg-[#3E4E68]  hover:bg-[#879AB8]  hover:scale-105 duration-500 text-center ml-auto flex flex-row justify-start items-center  gap-2 font-bold hover:text-Accent hover:border-Accent"
      type="submit"
    >
      Send Email
    </button>
  </form>
</div>
`;
